src/main.cpp

```diff
@@ -575,7 +574,169 @@ void affect_nearby_friends_with_alarm(struct Thing *traptng)
 
 long apply_wallhug_force_to_boulder(struct Thing *thing)
```

Comment:
This entire function does not look like the rewrite is finished yet.

Reply:
I don't see what else I can do to it.

Reply:
What Ed did to the other one from where you left off. As discussed on discord on 2021-07-09, and several times before,...

`One good rule of thumb they mentioned was that if you need to typecast, you most likely have the wrong variable type. So look on until you have the correct one.
and make it, clear, consistent, readable code.`

`There are also inlined functions. That sine and cosine table stuff is encapsulated in some function somewhere I think, just like the seed code.`

Reply:
What's left to do? I didn't think I'd be able to figure out the Sin and Cos stuff at the beginning, but I did!